packages/webapp/src/util/date.jsx

```diff
@@ -46,3 +46,15 @@ export const getLocalDateInYYYYDDMM = (date = new Date()) => {
   const [year, month, day] = [date.getFullYear(), date.getMonth() + 1, date.getDate()];
   return getDateInYYYYMMDD(year, month, day);
 };
+
```

Comment:
Is the `.jsx` extension here a holdover from a previous version of the file?

Reply:
Thank you for reviewing! I probably created the file by copying a component file. `useUnit`hook  still has `.jsx` too 😅 I will fix them in my next PR!